packages/pn-commons/src/components/TitleBox.tsx

```diff
@@ -50,6 +50,7 @@ const TitleBox: React.FC<Props> = ({
           variant={variantTitle}
           display="inline-block"
           sx={{ verticalAlign: 'middle' }}
+          aria-label={`${title}-page`}
```

Comment:
aria-labels on typography are wrongs, because the screen reader reads the text twice. Maybe the problem is on the typography variant

Reply:
in the windows os screen reader not read without that

Reply:
mmm it is not the correct way to fix the bug

Reply:
whitch is?

Reply:
try on your own